torch/_dynamo/variables/nn_module.py

```diff
@@ -74,7 +74,12 @@ def record_nn_module_stack(module_key: str, source, tx, mod: torch.nn.Module):
 
 
 class NNModuleVariable(VariableTracker):
-    _nonvar_fields = {"module_type", "module_key", "module", *VariableTracker._nonvar_fields}
+    _nonvar_fields = {
+        "module_type",
+        "module_key",
+        "module",
+        *VariableTracker._nonvar_fields,
+    }
```

Comment:
please don't reformat completely unrelated files on refactors like this, stuff like this is why I have to carefully review BE PRs from you lol